docs/uml-diagrams/FileManager.puml

```diff
@@ -0,0 +1,47 @@
+@startuml
+skinparam classAttributeIconSize 0
+
+class SuperTracker
+class FileManager
+class Inventory
+class Item
+class Command
+
+FileManager <.. SuperTracker
+Inventory <.. FileManager
+Item <. FileManager
+Item "*" <-* Inventory
+FileManager <.. Command
+Inventory <.. Command
```

Comment:
Should you use standard docorator to represent the class type such as `interface` or basic `class`.